chrononaut/__init__.py

```diff
@@ -24,33 +24,34 @@
 from chrononaut.exceptions import ChrononautException
 from chrononaut.flask_versioning import create_version
 from chrononaut.versioned import Versioned
+from chrononaut.models import activity_factory
+from chrononaut.migrations import MigrateFromHistoryTableOp, MigrateToHistoryTableOp
 
 
 def versioned_objects(items):
     for obj in items:
-        if hasattr(obj, "__history_mapper__"):
+        if hasattr(obj, "version"):
```

Comment:
We _may_ want some reserved property `__chrononaut_versioned__` or something just in case a model ever declares `.version` as a column.

Reply:
Makes sense! I'm glad you're looking into this, I'm going to have a careful look at the code again myself - the changes are pretty vast, so I could have missed something